main.go

```diff
@@ -4,27 +4,38 @@ import (
 	"fmt"
 	"github.com/gin-gonic/gin"
 	"github.com/joho/godotenv"
+	"html/template"
 )
 
+// Clarifai client and 500PX client singleton
+// To keep it simple, make them global so that we don't need to pass them around
+var CLFclient = NewClarifaiClient()
+//var PxClient = NewPxClient()  // TODO: later, if we want to post photos to 500px
+
 func main() {
+	// Create router
+	router := gin.Default()
 
-	app := gin.Default()
+	// TODO: Set up environment var (like prod/dev/test)
 
-	//load secret file
+	/* Here we load secret file for now. This should be removed later
+	 * The api secrets should be pre-set on the hosting machine and accessed via envVar during runtime
+         */
 	err := godotenv.Load()
 	if err != nil {
 		fmt.Print("Error loading .env file")
 	}
 
-	clarifai := NewClarifaiClient()
-	clarifai.RefreshAccesToken()
-
-	server := &Server{
-		app:      app,
-		clarifai: clarifai,
-	}
+	// Load static resources & templates
+	customTemplate := template.Must(template.New("main").ParseGlob("resources/templates/base/*.tmpl"))
+	customTemplate.ParseGlob("resources/templates/*.tmpl")
+	router.SetHTMLTemplate(customTemplate)
+	router.Static("/static", "resources/static")
 
-	server.Configure()
-	app.Run(":5050")
+	// Define routes **currently we only have two; it's not necessary to separate it to another file yet
+	router.GET("/", showHomePage)
+	router.POST("/getTags", fetchTags)
 
```

Comment:
no no I like how you got rid of `server.go`, it was probably an overkill for our use case. I was thinking more in the line of having helper functions (inside `main.go`) that will do some of those stuff